doc/reporters/github-reporter.md

```diff
@@ -2,6 +2,20 @@
 
 Report about design and static code diagnostics issues in pull requests based on GitHub Actions Workflow commands.
 
+1. Your git workflow must include/support flutter/dart
+2. Add to your git workflow:
+```
+jobs:
+  your_job_name:
+    ...
+    steps:
+     ...
+      - name: Run Code Metrics
+        run: flutter pub run dart_code_metrics:metrics lib --reporter=github lib
```

Comment:
This works if the project is added to `dev_dependencies` , but it seems that we should also cover a case when the project is installed as a global dependency.